packages/widgets/src/Transfer/Transfer.js

```diff
@@ -75,9 +64,9 @@ export const multiSelectedPropType = propTypes.arrayOf(singleSelectedPropType)
  * rest of the library
  */
 export const Transfer = ({
```

Comment:
I've been thinking about rendering custom options and how we could stay a little bit more consistent here with other components that may require customisation. It's quite difficult to come up with something that is exactly the same, since this `options` prop pattern is fundamentally different than the `children` pattern we use elsewhere. After chatting a little bit with @ismay I came up with an idea (not sure whether Ismay will fully agree).

I'll place a few comments in the component code to list the changes I'd like to suggest.